include/mbedtls/ssl.h

```diff
@@ -105,7 +105,7 @@
 #define MBEDTLS_ERR_SSL_WAITING_SERVER_HELLO_RENEGO       -0x6B00  /**< Unexpected message at ServerHello in renegotiation. */
 #define MBEDTLS_ERR_SSL_HELLO_VERIFY_REQUIRED             -0x6A80  /**< DTLS client must retry for hello verification */
 #define MBEDTLS_ERR_SSL_BUFFER_TOO_SMALL                  -0x6A00  /**< A buffer is too small to receive or write a message */
-#define MBEDTLS_ERR_SSL_NO_USABLE_CIPHERSUITE             -0x6980  /**< None of the common ciphersuites is usable (eg, no suitable certificate, see debug messages). */
+/* Error space gap */
```

Comment:
MBEDTLS_ERR_SSL_NO_CIPHER_CHOSEN needs to be removed as well (could be in another commit) and this commit (Remove MBEDTLS_ERR_SSL_NO_USABLE_CIPHERSUITE ) should update the documentation in error.h as done in the previous commits of this PR.